src/SFA.DAS.AssessorService.Application.Api.External/Controllers/StandardsController.cs

```diff
@@ -66,5 +66,27 @@ public async Task<IActionResult> GetOptionsForStandard([SwaggerParameter("Standa
 
             return Ok(requestedStandard);
         }
+
+        [HttpGet("options/{standardReference}/{version}")]
+        [SwaggerResponseExample((int)HttpStatusCode.OK, typeof(SwaggerHelpers.Examples.GetStandardVersionOptionsResponseExample))]
+        [SwaggerResponse((int)HttpStatusCode.OK, "The list of options.", typeof(StandardOptions))]
+        [SwaggerResponse((int)HttpStatusCode.NoContent, "The standard version was found, however it has no options.")]
+        [SwaggerResponse((int)HttpStatusCode.NotFound, "The standard version was not found.")]
+        [SwaggerOperation("Get Options for a standard version", "Gets the latest list of course options for the specified Standard version.", Produces = new string[] { "application/json" })]
+        public async Task<IActionResult> GetOptionsForStandardVersion(string standardReference, decimal version)
```

Comment:
Should version still be a string? otherwise ( down the line ) if they specify 1.10 in the query string, decimal makes that 1.1 which would be an older version?

Reply:
Agree version should be a string